### PR TITLE
[INT-423] Refactor Ethereum example connector

### PR DESCRIPTION
**Pull request by cnkarz**

### CODE REVIEW

**Review by rukolahasser — COMMENTED** [review]
Where is `pb.go`?

File: examples/ethereum/go.mod [hunk 1]
```diff
@@ -1,13 +1,11 @@
 module github.com/nakji-network/connector/examples/ethereum
 
-go 1.18
-
-replace github.com/nakji-network/connector => ../../
+go 1.19
```
[the remark this answers is not in the document]
Comment (cnkarz) [reply examples/ethereum/go.mod:3]: this is importing other modules so it is ok to upgrade this one

**Comment by cnkarz** [comment]
> Where is `pb.go`?

you added them to .gitignore? anyway I pushed it but there is no gitignore in compound example ass well as makefile etc.

File: examples/ethereum/ethereum.go [hunk 2]
```diff
@@ -7,114 +7,90 @@ package ethereum
...
 	"github.com/nakji-network/connector/kafkautils"
 
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/rs/zerolog/log"
 )
 
 type Connector struct {
-	*connector.Connector // embed Nakji connector.Connector into your custom connector to get access to all its methods
+	*ethereum.Connector // embed Nakji ethereum.Connector into your custom connector to get access to all its methods
```
Comment (rukolahasser) [inline examples/ethereum/ethereum.go:21]: It is confusing..

**Comment by rukolahasser** [comment]
> > Where is `pb.go`?
> 
> you added them to .gitignore? anyway I pushed it but there is no gitignore in compound example ass well as makefile etc.

I didn't add them to `.gitignore`

**Comment by cnkarz** [comment]
> I didn't add them to `.gitignore`

I thought you created these folders 😅 now I see it is older

File: examples/ethereum/ethereum.go (hunk 2 above)
Comment (cnkarz) [reply examples/ethereum/ethereum.go:21]: do you mean `ethereum` is confusing? how about `evm`?

Comment (rukolahasser) [reply examples/ethereum/ethereum.go:21]: `evm` looks good

**Review by rukolahasser — APPROVED** [review]

**Pull request merged**